cypress/integration/dashboard/dashboard_filtering_spec.js

```diff
@@ -12,234 +12,256 @@
 
 describe('dashboard filtering', () => {
   before(() => {
+    // WORK IN PROGRESS: https://github.com/AvivBenchorin/monterey/issues/11
+    cy.clearJSONMapping('cypress/fixtures/dashboard/data/mappings.json.txt')
 
-    // TO DO: Setup the indices and test data (see https://github.com/AvivBenchorin/monterey/issues/11)
-    // cy.request('PUT', 'localhost:9200/.kibana_1', cy.fixture('dashboard/mappings.json'))
-    // cy.request('POST', 'localhost:9200/.kibana_1', cy.fixture('dashboard/data.json.gz'))
+    cy.importJSONMapping('cypress/fixtures/dashboard/opensearch_dashboards/mappings.json.txt')
+    cy.importJSONMapping('cypress/fixtures/dashboard/data/mappings.json.txt')
+
+    cy.importJSONDoc('cypress/fixtures/dashboard/opensearch_dashboards/data.json.txt')
+    cy.importJSONDoc('cypress/fixtures/dashboard/data/data.json.txt')
   })
 
   after(() => {
-    // TO DO: Tear-down the indices and test data (see https://github.com/AvivBenchorin/monterey/issues/11)
-    // cy.request('DELETE', 'localhost:9200')
+    // WORK IN PROGRESS: https://github.com/AvivBenchorin/monterey/issues/11
+    cy.clearJSONMapping('cypress/fixtures/dashboard/data/mappings.json.txt')
```

Comment:
Shall the docs be cleaned as well?

Reply:
In the original `dashboard_filtering.js` test, only the indices had `DELETE` requests sent to OpenSearch (the rabbit-hole for finding this functionality starts [here](https://github.com/opensearch-project/OpenSearch-Dashboards/blob/9a2d3e6918760c4bba8b44b9e8dfcd14e459a916/test/functional/apps/dashboard/index.js#L43) and ends [here](https://github.com/opensearch-project/OpenSearch-Dashboards/blob/9a2d3e6918760c4bba8b44b9e8dfcd14e459a916/packages/osd-opensearch-archiver/src/lib/indices/delete_index.ts#L41)). My understanding is that deleting the OpenSearch index removes all of the data associated with it, and thus deleting doc by doc is unnecessary ([this](https://discuss.elastic.co/t/deleting-documents-vs-deleting-the-whole-index-and-rebuilding-it-periodically/127018) thread also suggests that deleting an index is more efficient than manually removing documents).